### ISSUE TITLE
CSS/Smarty Display Issue

### ISSUE THREAD

**Issue by WMGS** [formtools/core#70]
Uploaded files (binary) to server with FileZilla. Install screen normal, install successful. 

Login as admin after install and subsequent pages to not appear to utilize CSS or Smarty.
![image](https://user-images.githubusercontent.com/32196706/30743634-7174c280-9f6c-11e7-828a-420b1f95fa84.png)

Looking at page source and clicking on linked css files show the css code exists and available.

Attempted to change admin theme setting from default to Classic Grey and received error:

Notice: Undefined index: g_omit_top_bar in /home/mysite/public_html/cms2/themes/classicgrey/cache/42d5641a059249877f6c5e4d619abbd0b60352bd_0.file.header.tpl.php on line 91

Notice: Trying to get property of non-object in /home/mysite/public_html/cms2/themes/classicgrey/cache/42d5641a059249877f6c5e4d619abbd0b60352bd_0.file.header.tpl.php on line 91

Notice: Undefined index: footer_text in /home/mysite/public_html/cms2/themes/classicgrey/cache/6e56a033ec34535317f3ceaee8823790a4ae1cda_0.file.footer.tpl.php on line 45

Change admin settings back to default does not throw an error.

Deleted all and re-uploaded and re-installed twice. Same results.

**Comment by benkeen** [comment]
Thanks! Looks like the style tag isn't being properly included. If you still have the installation around, could you view source on any page that has the style messed up, like in your screenshot & post the HTML here? 

My hunch is there's a notice being thrown in the `<head>` causing the style tag to get out of whack. 

But curious it doesn't happen for me....

**Comment by benkeen** [comment]
Just reviewing the old Alpha bugs. I was never able to see this problem. If you have time, would you mind re-trying with the latest alpha and see if you're still seeing it? https://formtools.org/download/alpha.php

**Comment by WMGS** [comment]
Hi Ben,

 

I downloaded the newest alpha, re-installed, and same issue – no css loaded. Wiped out and tried again.

 

After clearing the brain fog, I realized all I needed to do was change $g_root_url = "http://... to $g_root_url = "https://... in the config file.

 

For some reason the install did not pickup (during getRootUrl?) that it was https so when it created the config file it was just http.

 

Not sure if it was just on my end but others might encounter this also.

 

Regards,

 

Bill

 

Bill Simon

Technical Coordinator

Florida Luxury Realty

727-755-3571
